cosmo/monitors/dark_monitors.py

```diff
@@ -72,32 +76,81 @@ def track(self):
+class FUVABottomDarkMonitor(FUVDarkMonitor):
+    """FUVA dark monitor for bottom edge"""
     data_model = FUVDarkDataModel
-    labels = ['ROOTNAME']
-    output = # path
-
+    segment = 'FUVA'
     location = (1060, 15250, 296, 375)
-    plottype = 'scatter'
-    x = 'date'
-    y = 'darks'
+    name = f'FUVA Dark Monitor - Bottom'
 
-    def get_data(self) -> Any:
-        filtered_rows = []
-        for _, row in self.model.new_data.iterrows():
-            if row.EXPSTART == 0:
-                continue
-            if row.SEGMENT == 'FUVA':
-                filtered_rows.append(dark_filter(row, True, self.location))
-        filtered_df = pd.concat(filtered_rows).reset_index(drop=True)
 
-        return explode_df(filtered_df, ['darks', 'date'])
+class FUVALeftDarkMonitor(FUVDarkMonitor):
+    """FUVA dark monitor for left edge"""
+    name = 'FUVA Dark Monitor - Left'
+    data_model = FUVDarkDataModel
+    segment = 'FUVA'
+    location = (1060, 1260, 296, 734)
 
-    def store_results(self):
-        # TODO: Define results to store
-        pass
 
-    def track(self):
-        # TODO: Define something to track
-        pass
+class FUVATopDarkMonitor(FUVDarkMonitor):
+    """FUVA dark monitor for top edge"""
+    name = 'FUVA Dark Monitor - Top'
+    data_model = FUVDarkDataModel
+    segment = 'FUVA'
+    location = (1060, 15250, 660, 734)
+
+
+class FUVARightDarkMonitor(FUVDarkMonitor):
+    """FUVA dark monitor for right edge"""
+    name = 'FUVA Dark Monitor - Right'
+    data_model = FUVDarkDataModel
+    segment = 'FUVA'
+    location = (15119, 15250, 296, 734)
+
+
+class FUVAInnerDarkMonitor(FUVDarkMonitor):
+    """FUVA dark monitor for inner region"""
+    name = 'FUVA Dark Monitor - Inner'
+    data_model = FUVDarkDataModel
+    segment = 'FUVA'
+    location = (1260, 15119, 375, 660)
+
+
+class FUVBBottomDarkMonitor(FUVDarkMonitor):
+    """FUVB dark monitor for bottom edge"""
+    name = 'FUVB Dark Monitor - Bottom'
+    data_model = FUVDarkDataModel
+    segment = 'FUVB'
+    location = (809, 15182, 360, 405)
+
+
+class FUVBLeftDarkMonitor(FUVDarkMonitor):
+    """FUVB dark monitor for left edge"""
+    name = 'FUVB Dark Monitor - Left'
+    data_model = FUVDarkDataModel
+    segment = 'FUVB'
+    location = (809, 1000, 360, 785)
+
+
+class FUVBTopDarkMonitor(FUVDarkMonitor):
+    """FUVB dark monitor for top edge"""
+    name = 'FUVB Dark Monitor - Top'
+    data_model = FUVDarkDataModel
+    segment = 'FUVB'
+    location = (809, 15182, 740, 785)
+
+
+class FUVBRightDarkMonitor(FUVDarkMonitor):
+    """FUVB dark monitor for right edge"""
+    name = 'FUVB Dark Monitor - Right'
+    data_model = FUVDarkDataModel
+    segment = 'FUVB'
+    location = (14990, 15182, 360, 785)
+
+
+class FUVBInnerDarkMonitor(FUVDarkMonitor):
+    """FUVB dark monitor for inner region"""
+    name = 'FUVB Dark Monitor - Inner'
+    data_model = FUVDarkDataModel
+    segment = 'FUVB'
+    location = (1000, 14990, 405, 740)
```

Comment:
The question of having individual monitors per "slice" is something that should probably be posed to whoever is providing science/instrumentation support to the monitors group. This is useful because it'd be trivial to run whichever ones of these that a user wants, and there can be customization on what's being tracked, plotted etc for the individual components, so on that front this is great. 

However, if it turns out that the quantity being tracked is something that comes from the "overall" FUVA Dark and the only thing desired from the "slices" are the figures, then it'd make more sense to have the one monitor with complex plots.